tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -985,53 +984,7 @@ trait WebActionsApiBaseTests extends ControllerTestCommon with BeforeAndAfterEac
       }
     }
 
-    it should s"handle http web action with base64 encoded known '+json' response (auth? ${creds.isDefined})" in {
```

Comment:
@rabbah, should these `+json` tests still be valid for raw web actions?

Reply:
they would be yes - I could change the test that’s removed to behave a raw web instead although note that I opened an issue to make the encoding just be a string in the future.

Reply:
don't think so - these are testing b64 action returns. can you clarify what are you thinking?

Reply:
Nevermind, I thought raw web actions were going to still return base64, but that doesn't seem to be the case.